worlds/smw/docs/setup_en.md

```diff
@@ -0,0 +1,150 @@
+# Super Mario World Randomizer Setup Guide
+
+## Required Software
+
+- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases). Make sure to check the box for `SNI Client - Super Mario World Patch Setup`
+
+
+- Hardware or software capable of loading and playing SNES ROM files
+    - An emulator capable of connecting to SNI such as:
+        - snes9x Multitroid
+          from: [snes9x Multitroid Download](https://drive.google.com/drive/folders/1_ej-pwWtCAHYXIrvs5Hro16A1s9Hi3Jz),
```

Comment:
🤷

Reply:
If this is a concern, it should probably be made an issue to be fixed across the codebase, but for now I'll do what the other SNES games do.

Reply:
This has already been changed in lttp and never existed in soe so I entirely disagree with this decision. Just because some other game did a bad thing doesn't mean you should too.

Reply:
The "xbrz" filter is the only GPL part of snes9x, if mulitroid has that filter then it's directly breaking license and really shouldn't be linke from us.
The Snes9x license further has:

```
  The copyright holders request that bug fixes and improvements to the code
  should be forwarded to them so everyone can benefit from the modifications
  in future versions.
```

While this is a request, not a requirement I'm almost certain it wasn't fulfilled either.

Reply:
If this is that much of a problem, then I'll just remove it.
I don't know anything about the current state of the setup process for SNES games for AP. I haven't set up an emulator persuant of use with a multiworld in over two years. I assumed that what existed in the codebase already wrt setup documentation was valid and accepted (otherwise why would it have passed review to begin with?). If that's not the case, then that should be fixed, but don't blame me for following precedent.